src/test/java/carracing/domain/race/RaceTest.java

```diff
@@ -0,0 +1,82 @@
+package carracing.domain.race;
+
+import carracing.domain.car.Car;
+import carracing.domain.car.Name;
+import carracing.domain.car.Position;
+import carracing.domain.move.ForwardStrategy;
+import carracing.domain.move.MoveStrategy;
+import carracing.domain.move.StopStrategy;
+import carracing.domain.record.RoundRecord;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RaceTest {
+
+    @DisplayName("이름 리스트로 자동차들을 초기화 할 수 있다")
+    @Test
+    void of() {
+        MoveStrategy moveStrategy = new StopStrategy();
+        List<String> carNames = List.of("green", "blue", "red");
+        List<Car> expected = List.of(
+                Car.of(Name.from("green"), Position.from(1)),
+                Car.of(Name.from("blue"), Position.from(1)),
+                Car.of(Name.from("red"), Position.from(1))
+        );
+
+        Race race = Race.of(carNames, 1, moveStrategy);
+
+        assertThat(race.getCars())
+                .hasSize(3)
+                .usingRecursiveComparison()
+                .isEqualTo(expected);
+    }
+
+    @DisplayName("난수값이 4이상이면 자동차를 1만큼 움직인다.")
+    @ParameterizedTest
+    @ValueSource(ints = {4, 5, 9, 100})
+    void moveCars(int randomNumber) {
+        MoveStrategy moveStrategy = new ForwardStrategy();
+        List<String> carNames = List.of("green");
+        Race race = Race.of(carNames, 1, moveStrategy);
```

Comment:
인터페이스가 단일 메소드만 가지고 있어 함수형 인터페이스로 볼 수 있는데요
이때 람다식을 사용하면 클래스 구현 없이도 표현할 수 있습니다.

```suggestion
        Race race = Race.of(carNames, 1, () -> true);
```
참고만 해주세요~

Reply:
전혀 생각도 못한 부분인데, 추후 미션에서 사용할 일 있으면 적용해보겠습니다👍